<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab_main_window">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>-1</y>
      <width>931</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Main window</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Account details</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>User ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>183</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Charities</string>
      </property>
     </widget>
     <widget class="QListWidget" name="list_widget_ad_charities">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>321</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>183</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Events</string>
      </property>
     </widget>
     <widget class="QListWidget" name="list_widget_ad_events">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>210</y>
        <width>331</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ad_withdraw_event">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>520</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Withdraw from event</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_ad_user_id">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_ad_password">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ad_change_user_id">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>20</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Change User ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ad_change_password">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>100</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_ad_unavailable_user_id">
      <property name="geometry">
       <rect>
        <x>136</x>
        <y>63</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string>color: red;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_ad_unauthorized_password">
      <property name="geometry">
       <rect>
        <x>136</x>
        <y>143</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string>color: red;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ad_cancel_changes">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>60</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel changes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ad_confirm_changes">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>60</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm changes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ad_remove_account">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>570</y>
        <width>921</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove account</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_logout">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>650</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Log out</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
